hack/verify-generated-docs.sh

```diff
@@ -16,6 +16,10 @@ GENERATED_DOCS_ROOT="${OS_ROOT}/${GENERATED_DOCS_ROOT_REL}"
 TMP_GENERATED_DOCS_ROOT_REL="_output/verify-generated-docs"
```

Comment:
@smarterclayton 
>These tasks need to be added to make verify and make update or they won't be run.

The `hack/verify-generated-docs.sh` task was are already there from previous contributions, I simply updated it so that it verifies man pages as well